source/release-notes/3.0.txt

```diff
@@ -390,14 +390,22 @@ MongoDB 3.0 includes the following query enhancements:
 
 .. seealso:: :ref:`3.0-geo-near-compatibility`
 
-Distributions
-~~~~~~~~~~~~~
+Distributions and Supported Versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Most non-Enterprise MongoDB distributions now include support for SSL.
 Previously, only MongoDB Enterprise distributions came with SSL support
 included; for non-Enterprise distributions, you had to build MongoDB
 locally with the ``--ssl`` flag (i.e. ``scons --ssl``).
 
+32-bit MongoDB builds are available for testing, but are not for
+production use. 32-bit MongoDB builds do not include the WiredTiger
+storage engine.
+
+MongoDB builds are available for Windows Server 2003 and Windows Vista
+(as "64-bit Legacy"), but the minimum officially supported Windows
+versions are Windows Server 2008 and later.
+
```

Comment:
could you add a link to http://docs.mongodb.org/manual/release-notes/3.0-compatibility/#platform-support and probably http://docs.mongodb.org/manual/faq/fundamentals/#what-are-the-limitations-of-32-bit-versions-of-mongodb